astrid/plugin-src/com/todoroo/astrid/gcal/CalendarAlarmListCreator.java

```
@@ -0,0 +1,230 @@
+                    tagData.setValue(TagData.MEMBER_COUNT, membersArray.length());
+                    tagDataService.save(tagData);
+                    dismissListener.onClick(v);
+                }
+            }
+        });
+
+        moreOptions.setOnClickListener(new OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(CalendarAlarmListCreator.this, TaskListActivity.class);
+                intent.putExtra(TaskListFragment.TOKEN_NEW_LIST_NAME, tagName);
+                intent.putExtra(TaskListFragment.TOKEN_NEW_LIST_MEMBERS, buildMembersArray().toString());
+                intent.putExtra(TaskListFragment.TOKEN_NEW_LIST, true);
+                startActivity(intent);
+                dismissListener.onClick(v);
+            }
+        });
+    }
+
+    private String buildAttendeesString() {
+        if (emails.size() == 0) {
+            return ""; //$NON-NLS-1$
+        } else if (emails.size() == 1) {
+            String displayName = getDisplayName(0);
+            return getString(R.string.CRA_one_attendee, displayName);
+        } else { // emails.size() >= 2
+            String displayName1 = getDisplayName(0);
+            String displayName2 = getDisplayName(1);
+
+            int extras = emails.size() - 2;
+            if (extras > 0)
+                return getString(R.string.CRA_many_attendees, displayName1, displayName2, extras);
+            else
+                return getString(R.string.CRA_two_attendees, displayName1, displayName2);
+
+        }
+    }
+
+    private JSONArray buildMembersArray() {
+        JSONArray array = new JSONArray();
+        for (String email : emails) {
+            JSONObject member = new JSONObject();
+            try {
+                member.put("email", email); //$NON-NLS-1$
+                array.put(member);
+            } catch (JSONException e) {
+                //
```

you should say something intellegent if you're going to suppress errors... e.g. "skip this member, keep going". or log the error.